rightmesh/contracts/MeshCrowdsale.sol

```diff
@@ -59,14 +59,28 @@ contract MeshCrowdsale is CappedCrowdsale, Ownable {
 
   /**
    * @dev Allows the current owner to update contribution limits
-   * @param _address whose contribution limits should be changed
+   * @param _addresses whose contribution limits should be changed
    * @param _weiLimit new contribution limit
    * @return boolean indicating function success.
    */
-  function setLimit(address _address, uint256 _weiLimit) external onlyOwner returns (bool) {
-    // only allow changing the limit to be greater than current contribution
-    require(_weiLimit >= weiContributions[_address]);
-    weiLimits[_address] = _weiLimit;
+  function setLimit(address[] _addresses, uint256 _weiLimit) external onlyOwner returns (bool) {
+    for (uint i = 0; i < _addresses.length; i++) {
+      address _address = _addresses[i];
+
+      // only allow changing the limit to be greater than current contribution
+      require(_weiLimit >= weiContributions[_address]);
+      weiLimits[_address] = _weiLimit;
+    }
+    return true;
```

Comment:
What is the expected behaviour when some addresses have contributed over _weiLimit and some haven't?

This true doesn't really tell you which addresses were updated (maybe that is ok?). Is it possible to return an array of trues for which accounts in the batch succeeded and which didn't?